xml/deployment_hwrecommend.xml

```diff
@@ -375,11 +375,13 @@
   <title>Network Recommendations</title>
 
   <para>
-   The network environment where you intend to run &ceph; should ideally be a
-   bonded set of at least two network interfaces that is logically split into a
-   public part and a trusted internal part using VLANs. The bonding mode is
-   recommended to be 802.3ad if possible to provide maximum bandwidth and
-   resiliency.
+   For the &ceph; cluster network environment, we recommend either two bonded
+   10 GbE network interfaces, or one 25 GbE network interface. The network
```

Comment:
Is that our recommendation? I would start with at least 25GbE bonded network and given the price of 100GbE in the meanwhile everyone should aim for that in a "larger" enviroment and also if we talk about all-flash in the meanwhile. 

Someone could of course also use 10GbE but in 2019 I think we should at least start with 25GbE not only because of the bandwith but foremost because of the latency.

Reply:
You're right, we " required 4x 10 GbE, recommended 2x 25 GbE" - see https://documentation.suse.com/ses/6/single-html/ses-deployment/#sysreq-osd. Going with 100 GbE is a good recommendation but needs more work overall.

Reply:
@tbazant - Kai has confirmed this is accurate.